framework/test/jest/unit/specs/modules/chain/dpos/getRoundDelegates/getRoundDelegates.spec.js

```diff
@@ -71,37 +73,18 @@ describe('dpos.getRoundDelegates()', () => {
 	});
 
 	describe('Given the round is NOT in the round_delegates table', () => {
-		let list;
 		beforeEach(async () => {
 			// Arrange
-			when(stubs.storage.entities.RoundDelegates.getRoundDelegates)
+			when(stubs.storage.entities.RoundDelegates.getActiveDelegatesForRound)
 				.calledWith(roundNo)
 				.mockResolvedValue([]);
 			stubs.storage.entities.Account.get.mockResolvedValue(delegateAccounts);
-
-			// Act
-			list = await dpos.getRoundDelegates(roundNo);
-		});
-
-		it('should return shuffled delegate list by using delegate accounts', () => {
-			// Assert
-			expect(stubs.storage.entities.Account.get).toHaveBeenCalledWith(
-				{ isDelegate: true },
-				{
-					limit: constants.ACTIVE_DELEGATES,
-					sort: ['voteWeight:desc', 'publicKey:asc'],
-				},
-			);
-			expect(list).toEqual(shuffledDelegatePublicKeys);
```

Comment:
please move this test scenario to "apply.spec.js" since it's actually part of apply behaviour.

Reply:
This test has been removed as an error should be thrown when the round delegates table is not in the db?